examples/MuEvGen/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# - Basic CMake setup
+# Check version meets your requirements
+# Declare project, which will configure compiler for us
+cmake_minimum_required(VERSION 2.8)
+project(MuEvGen)
+
+set(CMAKE_BUILD_TYPE "Release")
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# c++11 switch
+list(APPEND CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
This can be removed - the use of the `CMAKE_CXX_STANDARD` variable(s) above will automatically add the required flags to the compilation commands.